ivy/functional/frontends/jax/lax/operators.py

```diff
@@ -32,3 +32,10 @@ def sqrt(x):
      
 
 sqrt.unsupported_dtypes = {"torch": ("float16", "bfloat16")}
+
+
+def qr(x, *, full_matrices=True):
+    return ivy.qr(x, "complete" if full_matrices else "reduced")
```

Comment:
mode is a positional argument, instead of writing the conditional statement directly in the argument try storing its result in a variable and then passing it.